s4e-web/src/app/views/map-view/timeline/timeline.component.ts

```diff
@@ -126,5 +145,17 @@ export class TimelineComponent implements OnInit, OnDestroy {
       .subscribe(() => this.hackCalendar());
   }
 
-  ngOnDestroy(): void {}
+  ngOnDestroy(): void {
+  }
+
+  ngOnChanges(changes: SimpleChanges): void {
+    if (changes.resolution || changes.startTime) {
+      this.hourmarks = [];
+      for (let i = 0; i < this.resolution && this.hourmarks.length < TimelineComponent.HOURMARKS_COUNT; i += (this.resolution / TimelineComponent.HOURMARKS_COUNT)) {
```

Comment:
Wouldn't it be more efficient to store this equitation in variable `this.resolution / TimelineComponent.HOURMARKS_COUNT`?